gunicorn/config.py

```diff
@@ -1121,6 +1122,18 @@ class Loglevel(Setting):
         """
 
 
+class CaptureSTDIO(Setting):
+    name = "capture_stdio"
+    section = "Logging"
+    cli = ["--capture-stdio"]
```

Comment:
IMO IO is a bit confusing since it doesn't do anything with stdin. CPython has a sort of similar (it temporarily redirects stdout and stderr to an io.StringIO instance) function named captured_output: https://github.com/python/cpython/blob/master/Lib/test/support/__init__.py#L1392